Makefile

```diff
@@ -23,6 +27,73 @@ include deployment/*/Makefile
 include dashboards/Makefile
 include package/release/Makefile
 
+define RELEASE_GOOS_GOARCH
```

Comment:
Let's don't use advanced Makefile constructs, since this complicates further maintenance of these constructs:

- a few people know Makefile basics
- nobody likes debugging Makefile rules
- nobody knows advanced Makefile constructs
- advanced Makefile constructs can work differently across the supported development platforms (various versions of Linux, MacOS, Windows, etc.)

So let's try sticking with the most simple Makefile constructs only.